src/components/drivers/photo-viewer.jsx

```diff
@@ -25,23 +25,24 @@ export default class PhotoViewer extends Component {
     return { height: imgHeight, width: imgWidth };
   }
 
+  componentDidMount() {
+    const { originalWidth, originalHeight } = this.props;
+    const imageDimensions = this.getImageDimensions.call(this, originalWidth, originalHeight);
+
+    this.props.texture.image.style.width = `${imageDimensions.width}px`;
+    this.props.texture.image.style.height = `${imageDimensions.height}px`;
+    this.props.texture.image.setAttribute('class', 'photo');
+    document.getElementById('pg-photo-container').appendChild(this.props.texture.image);
```

Comment:
we're really piercing react a lot in these components. probably something we want to revisit